src/components/AddStudent.js

```diff
@@ -0,0 +1,67 @@
+import * as React from "react";
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+ 
+class AddStudent extends React.Component {
+  constructor(props) {
+    super(props);
+    //Neccessary information to post and communicate errors/success
+    this.state={ name:'',uEmail:'', message:'' };
+  }
+  
+  
+  handleSubmit = () => {
```

Comment:
If you want, the `async await` syntax is pretty useful for this kind of thing, and can really shorten a method like this.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function

Reply:
Thanks for the heads up.